src/defaultTranslations/relative.time.ts

```diff
@@ -265,6 +265,35 @@ export const translations: DefaultTranslations = {
       year_in_plural: 'あと __count__ 年'
     }
   },
+  lt: {
+    translation: {
+      now: 'šiuo metu',
+      second_ago: 'prieš __count__ sekundę',
+      second_ago_plural: 'prieš __count__ sekundes',
+      second_in: 'po __count__ sekundės',
+      second_in_plural: 'po __count__ sekundžių',
+      minute_ago: 'prieš __count__ minutę',
+      minute_ago_plural: 'prieš __count__ minutes',
+      minute_in: 'po __count__ minutės',
+      minute_in_plural: 'po __count__ minučių',
+      hour_ago: 'prieš __count__ valandą',
+      hour_ago_plural: 'prieš __count__ valandas',
+      hour_in: 'po __count__ valandos',
+      hour_in_plural: 'po __count__ valandų',
+      day_ago: 'prieš __count__ dieną',
+      day_ago_plural: 'prieš __count__ dienas',
+      day_in: 'po __count__ dienos',
+      day_in_plural: 'po __count__ dienų',
+      month_ago: 'prieš __count__ mėnesį',
+      month_ago_plural: 'prieš __count__ mėnesius',
+      month_in: 'po __count__ mėnesio',
+      month_in_plural: 'po __count__ mėnesių',
+      year_ago: 'prieš __count__ metus',
+      year_ago_plural: 'prieš __count__ metus',
+      year_in: 'po __count__ metų',
```

Comment:
`po` is `after` whereas we here need `in`.
So maybe `per __count__ metų` instead?

Reply:
From what I have seen (I used the german translation, because that is my first language), `per` means `in` in the sense of `within`, and not `in` as in `after`. 
(I hope that the meaning is clear). So in this case I would go with `po __count__ metų`.

Reply:
yeah but the `year_in` is exactly saying that `in` with the meaning of `within` if you look at the german/english translation. `after` would be exactly what `ago` expresses.

Reply:
So like if you visit a blog post and it says "last visited 2 years ago" that be year_ago whereas "concert tickets can be bought in 1 year", means now + 1 year

Reply:
Alright, this is not a hill I'm willing to die on ;). But this would mean changing all `*_in` instances from `po` to `per`, right?

Reply:
Tbh I have no idea which is the better one and I'd be second guessing. To keep things sane, how about we get this merged and if somebody has a feeling it's not right they can open up another PR.